tests/org.eclipse.n4js.transpiler.tests/src/org/eclipse/n4js/transpiler/sourcemap/Base64VLQTestMultipleNumbers.java

```diff
@@ -63,11 +65,19 @@ public void testEncode() {
 	}
 
 	/**
-	 * Test method for {@link org.eclipse.n4js.transpiler.sourcemap.Base64VLQ#decode(java.lang.String)}.
+	 * Test method for {@link org.eclipse.n4js.transpiler.sourcemap.Base64VLQ#decode(java.lang.CharSequence)}.
 	 */
 	@Test
 	public void testDecode() {
 		assertArrayEquals(decimals, Base64VLQ.decode(base64VLQ));
 	}
 
+	/**
+	 * Test en/decoding
+	 */
+	@Test
+	public void testRoundTrip() {
+		assertEquals(base64VLQ, Base64VLQ.encode(Base64VLQ.decode(base64VLQ)));
+	}
+
```

Comment:
This test class and `Base64VLQTestSingleNumbers` are currently not executed in maven since they are missing the mandatory `*Test` suffix.